### ISSUE TITLE
textstat_simil: n argument returns first n documents/features, not highest n

### ISSUE THREAD

**Issue by thomasd2** [kbenoit/quanteda#754]
Adapted example from help file, simplified to one selected term:
```
# compute some term similarities
(s2 <- textstat_simil(presDfm, "fair", method = "cosine", 
                      margin = "features", n = 8))
as.list(s2)
# these are the same as the first 8 features:
setdiff(names(as.list(s2)$fair),featnames(presDfm)[1:8])
(s3 <- textstat_simil(presDfm, "fair", method = "cosine", 
                      margin = "features", n = 15))
as.list(s3)
# these are the same as the first 15 features:
setdiff(names(as.list(s3)$fair),featnames(presDfm)[1:15])
```

**Comment by HaiyanLW** [comment]
Thanks. What's the version of `quanteda` you are using? I believe this is an old issue which has been fixed. You might want to update your `quanteda`.

**Comment by thomasd2** [comment]
Thanks. It's `quanteda_0.9.9-50`, so that should be the latest, or are you referring to a development version?

**Comment by kbenoit** [comment]
Fixed in https://github.com/kbenoit/quanteda/commit/a11fa2d5e6fa7d717bb5c8dd76f961a49a51cc18#diff-33d9ca6a4294b3e11db0b8eb6a937279. 

CRAN refresh is pending right now, but it's fixed in the GitHub version (currently v0.9.9.65).
